tensorflow/python/data/experimental/kernel_tests/bucket_by_sequence_length_test.py

```diff
@@ -72,10 +74,141 @@ def _get_record_shape(sparse):
 
 
 @test_util.run_all_in_graph_and_eager_modes
-class BucketBySequenceLengthTest(test_base.DatasetTestBase):
+class BucketBySequenceLengthTest(test_base.DatasetTestBase, parameterized.TestCase):
+
+  @parameterized.named_parameters(
+    ("WithoutPadding", True),
+    ("WithPadding", False),
+  )
+  def testBucketDropReminder(self, param_no_padding):
```

Comment:
make this testSkipEagerBucketDropReminder for now and add a TODO note for eager coverage.